src/db_impl.cc

```diff
@@ -1398,17 +1407,20 @@ void TitanDBImpl::OnCompactionCompleted(
           // Mark last two level blob files to merge in next compaction if
           // discardable size reached GC threshold
           if (file->NoLiveData()) {
+            RecordTick(statistics(stats_.get()), TITAN_GC_NUM_FILES, 1);
+            // TODO: use another tick type instead of GC_SMALL_FILE
+            RecordTick(statistics(stats_.get()), TITAN_GC_SMALL_FILE, 1);
             edit.DeleteBlobFile(file->file_number(),
                                 db_impl_->GetLatestSequenceNumber());
           } else if (static_cast<int>(file->file_level()) >=
                          cf_options.num_levels - 2 &&
                      file->GetDiscardableRatio() >
                          cf_options.blob_file_discardable_ratio) {
+            // TODO: use another tick type instead of GC_DISCARDABLE
+            RecordTick(statistics(stats_.get()), TITAN_GC_DISCARDABLE, 1);
```

Comment:
Old usage of `TITAN_GC_SMALL_FILE` and `TITAN_GC_DISCARDABLE` should be removed.